File: s08e02_mapas/mapas.py
        self.age: int = age
    def __str__(self):
        return self.name + ":" + str(self.age)

pessoas: List[Pessoa] = [Pessoa("ana", 5), Pessoa("bia", 2), Pessoa("caio", 7)]

#print(pessoas[2]) # acesso aleatório

apelidos: Dict[str, Pessoa] = {} # dict key value
apelidos["vivi"] = Pessoa("David", 36)
apelidos["cabiludo"] = Pessoa("Wermyson", 18)
apelidos["harrypotter"] = Pessoa("Joas", 20)
apelidos["travesseiro"] = Pessoa("Leticia", 18)
apelidos["lagartixa"] = Pessoa("Luis", 15)

for key in apelidos.keys():
    print(key, end=" ")
print()

for pessoa in apelidos.values():
    print(pessoa, end=" ")
print()

for key in apelidos:
    print(key, apelidos[key])

File: s08e02_mapas/Mapas.java
// import java.security.KeyStore.Entry;
// import java.util.ArrayList;
// import java.util.LinkedList;
// import java.util.List;
// import java.util.Map;
// import java.util.TreeMap;

// class Pessoa {
//     String nome;
//     Pessoa(String nome){
//         this.nome = nome;
//     }
//     public String toString() {
//         return nome;
//     }
// }

//  public class Mapas {
//     public static void main(String[] args) {
//         List<Pessoa> pessoas = new ArrayList<Pessoa>();
//         pessoas.add(new Pessoa("Vanessa"));

//         Map<String, Pessoa> apelidos = new TreeMap<>();
//         apelidos.put("vivi", new Pessoa("David"));
//         apelidos.put("lagartixa", new Pessoa("Luis"));
//         apelidos.put("travesseiro", new Pessoa("Leticia"));
//         apelidos.put("cabiludo", new Pessoa("Wermyson"));

//         System.out.println(apelidos);

//         if(apelidos.containsKey("vivi")){
//             System.out.println("Essa pessoa ja esta cadastrada");
//         }
//         for(String key : apelidos.keySet())
//             System.out.println(key);

//         for(Pessoa pessoa: apelidos.values())
//             System.out.println(pessoa);

//         for(Map.Entry<String, Pessoa> entry : apelidos.entrySet())
//             System.out.println(entry.getKey() + ":" + entry.getValue());
//     }
// }